src/log-pcap.c

```diff
@@ -424,9 +424,7 @@ static int PcapLogOpenHandles(PcapLogData *pl, const Packet *p)
     int datalink = p->datalink;
     if (IS_TUNNEL_PKT(p) && !IS_TUNNEL_ROOT_PKT(p)) {
         Packet *real_p = p->root;
-        SCMutexLock(&real_p->tunnel_mutex);
```

Comment:
What about around line 619 ?
Coverity issued a warning about this

Reply:
Good catch. Removing all.